lxc/driver.go

```diff
@@ -47,6 +47,10 @@ var (
 			hclspec.NewLiteral("true"),
 		),
 		"lxc_path": hclspec.NewAttr("lxc_path", "string", false),
+		"destroy_containers": hclspec.NewDefault(
```

Comment:
The Docker driver bundles up this concept under the [`gc`](https://www.nomadproject.io/docs/drivers/docker.html#gc) block, which would provide some flexibility for adding other GC mechanisms in the future. Let's try to mirror that if we can.

Reply:
@tgross  see latest commits. I also added new tests for GC.Container=true and GC.Container=false.